app/scripts/configs/default-tracks-for-datatype.js

```diff
@@ -22,7 +22,7 @@ export const DEFAULT_TRACKS_FOR_DATATYPE = {
   'geo-json': {
     center: 'geo-json',
   },
-  'gene-annotations': {
+  'gene-annotation': {
```

Comment:
Would you mind changing the docs  to this as well? https://github.com/higlass/higlass/blob/develop/docs/track_types.rst#gene-annotations

We must have let that inconsistency slide for a while.